File: Database/genConvert/MultiTowerRewardItem.py
def MultiTowerRewardItem(json):
    this={}#MultiTowerRewardItemjson)
    #thrownewInvalidJSONException()
    if 'round_st' in json:
        this['round_st'] = json['round_st']
    if 'round_ed' in json:
        this['round_ed'] = json['round_ed']
    if 'itemname' in json:
        this['itemname'] = json['itemname']
    if 'num' in json:
        this['num'] = json['num']
    if 'type' in json:
        this['type'] = json['type']
return this

File: Database/Assembly_SRPG_JP/MultiTowerRewardItem.cs
// Decompiled with JetBrains decompiler
// Type: SRPG.MultiTowerRewardItem
// Assembly: Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: 85BFDF7F-5712-4D45-9CD6-3465C703DFDF
// Assembly location: S:\Desktop\Assembly-CSharp.dll

namespace SRPG
{
  public class MultiTowerRewardItem
  {
    public int round_st;
    public int round_ed;
    public MultiTowerRewardItem.RewardType type;
    public string itemname;
    public int num;

    public void Deserialize(JSON_MultiTowerRewardItem json)
    {
      if (json == null)
        throw new InvalidJSONException();
      this.round_st = json.round_st;
      this.round_ed = json.round_ed;
      this.itemname = json.itemname;
      this.num = json.num;
      this.type = (MultiTowerRewardItem.RewardType) json.type;
    }

    public enum RewardType : byte
    {
      None,
      Item,
      Coin,
      Artifact,
      Award,
      Unit,
      Gold,
    }
  }
}
